test/units/test_token_store.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require 'jwt_sessions'
+
+class TestTokenStore < Minitest::Test
+  def teardown
+    JWTSessions.remove_instance_variable(:@token_store) if JWTSessions.instance_variable_defined?(:@token_store)
+  end
+
+  def test_setting_redis_token_store_by_default
+    assert_instance_of JWTSessions::StoreAdapters::RedisStoreAdapter, JWTSessions.token_store
+  end
+
+  def test_setting_redis_token_store_with_default_prefix
+    JWTSessions.token_store = :redis, { redis_url: 'redis://127.0.0.1:6379/0' }
+    assert_instance_of JWTSessions::StoreAdapters::RedisStoreAdapter, JWTSessions.token_store
+    assert_equal 'jwt_', JWTSessions.token_store.prefix
+  end
+
+  def test_setting_redis_token_store
+    JWTSessions.token_store = :redis, { redis_url: 'redis://127.0.0.1:6379/0', token_prefix: 'prefix' }
+    assert_instance_of JWTSessions::StoreAdapters::RedisStoreAdapter, JWTSessions.token_store
+    assert_equal 'prefix', JWTSessions.token_store.prefix
+  end
+
+  def test_setting_redis_token_store_along_with_module_configuration
+    JWTSessions.redis_port = 6378
+    JWTSessions.token_store = :redis
+
+    assert_equal 'redis://127.0.0.1:6378/0', JWTSessions.token_store.redis_client.connection[:id]
```

Comment:
redis port should be reset on the teardown

Reply:
Good catch, fixed that